Makefile

```diff
@@ -31,6 +31,7 @@ DOCKER_BUILD = docker build --no-cache \
 	    -f slave.mk \
 	    PLATFORM=$(PLATFORM) \
 	    SKU=$(SKU) \
+            DEBUG_BUILD=$(DEBUG_BUILD) \
```

Comment:
in the rules/config file, and a build option
DEBUG_BUILD = n

by default, we do not enable debug_build. You can also change it to DEBUG_BUILD=y